CMakeLists.txt

```diff
@@ -72,9 +103,13 @@ FetchContent_MakeAvailable(Universal_Robots_Client_Library)
 FetchContent_Declare(
   viam-cpp-sdk
   GIT_REPOSITORY https://github.com/viamrobotics/viam-cpp-sdk
-  GIT_TAG releases/v0.13.2 # Keep in sync with Dockerfile
+  GIT_TAG releases/v0.19.0 # Keep in sync with Dockerfile
   GIT_SHALLOW TRUE
   SYSTEM
+  # Without this patch command, using this as a SYSTEM package results
+  # in failed compilation due to the statically included protobuf gens
+  # getting in the wrong order. Just delete them, we don't need them.
+  PATCH_COMMAND find ./src/viam/api -name "*.pb.h" -type f -exec rm {} +
```

Comment:
Ultimately, I'd like the C++ SDK to drop their pre-built proto files, but for now I'm doing it manually.